lib/rules/no-arrow-tests.js

```diff
@@ -62,12 +62,7 @@ module.exports = {
 
             const beforeArrowComment = extractSourceTextByRange(beforeArrowToken.range[1], arrow.range[0]);
             const afterArrowComment = extractSourceTextByRange(arrow.range[1], fn.body.range[0]);
-            let paramsFullText;
-            if (firstToken.type !== "Punctuator") {
-                paramsFullText = `(${params}${beforeArrowComment})${afterArrowComment}`;
-            } else {
-                paramsFullText = `${params}${beforeArrowComment}${afterArrowComment}`;
-            }
+            const paramsFullText = firstToken.type !== "Punctuator" ? `(${params}${beforeArrowComment})${afterArrowComment}` : `${params}${beforeArrowComment}${afterArrowComment}`;
```

Comment:
I don't love the readability loss here. Can we at least add line breaks?

Reply:
Added.